<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>downfw_config</class>
 <widget class="QDialog" name="downfw_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>601</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Lưu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>520</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Xoá</string>
   </property>
  </widget>
  <widget class="QLabel" name="mainApp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Main App</string>
   </property>
  </widget>
  <widget class="QPushButton" name="linkFileMainApp">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>177</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Chọn file </string>
   </property>
  </widget>
  <widget class="QLabel" name="linkFile_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="linkFile">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>130</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Chọn file</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameDevice">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>33</y>
     <width>125</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="codeDevice">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>65</y>
     <width>157</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Device select</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BP - MainApp</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BP - Bootloader 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BP - Bootloader 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HMI - MainApp</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HMI - Bootloader 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HMI - Bootloader 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MC - MainApp</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MC - Bootloader 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MC - Bootloader 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PMU - MainApp</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PMU - Bootloader 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PMU - Bootloader 1</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="version">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>98</y>
     <width>125</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>99</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Phiên Bản</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>33</y>
     <width>69</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tên Thiết Bị</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>66</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Mã Thiết Bị</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>132</y>
     <width>77</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>File Firmware</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
